measure.sh

```diff
@@ -6,7 +6,7 @@ url=http://localhost:8001/api/v1/namespaces/monitoring/services/monitoring-kube-
 trap 'jobs -p | xargs -r kill -TERM' INT TERM EXIT HUP
 
 # open the proxy in the background
-kubectl proxy --port=8001 &
+kubectl proxy &
```

Comment:
:question: If port 8001 was already occupied, then querying `localhost:8001` would also be wrong, right?